oaknational/Oak-Web-Application
# Feat/nav bar

## Description

- Change order of links in the nav bar
- Position of arrow changes based on the current pathname in browser

## Issue(s)

Fixes #460 

## How to test

1. Go to [preview here](https://oak-web-application-git-feat-nav-bar-oak-national-academy.vercel.app/)
2. Click on Hamburger icon to open the nav bar.
3. All links in the nav bar should be working. You should see the position of the arrow in the nav bar change based on the current page you are on.

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:

## Checklist
